draft-ietf-tls-tls13.md

```diff
@@ -2743,7 +2743,8 @@ When this message will be sent:
 
 > A non-anonymous server can optionally request a certificate from the client,
 if appropriate for the selected cipher suite. This message, if sent, will
-immediately follow the server's Certificate message.
+immediately follow the server's Certificate message. The server may also send
+this message at any time after the handshake is complete.
```

Comment:
Didn't we agree that the client would be permitted to authentication either zero or one times?  This needs to call that out.